pkg/obfuscator/domain.go

```diff
@@ -4,18 +4,21 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+
+	"github.com/openshift/must-gather-clean/pkg/schema"
 )
 
 const (
-	domainPattern      = `([a-zA-Z0-9\.]*\.)?(%s)`
-	obfuscatedTemplate = "domain%07d"
+	domainPattern           = `([a-zA-Z0-9\.]*\.)?(%s)`
+	obfuscatedTemplate      = "domain%07d"
+	staticDomainReplacement = "obfuscated.com"
 )
 
 type domainObfuscator struct {
 	ReplacementTracker
-	domainCount    int
-	domainPatterns []*regexp.Regexp
-	domainMapping  map[string]string
+	replacementType schema.ObfuscateReplacementType
```

Comment:
why do we need that here? that should be in the generator

Reply:
Addressed